src/Components/Search/NewSearch/Mobile/Overlay.tsx

```diff
@@ -67,6 +77,8 @@ export const Overlay: FC<OverlayProps> = ({ viewer, relay, onClose }) => {
           }
 
           scrollToTop()
+          // trigger useEffect to send tracking event
+          setFetchCounter(prevCounter => prevCounter + 1)
```

Comment:
Just pass a callback and use that for tracking. The state is unnecessary — in general avoid effects if you can.

Reply:
solved!